eli5/xgboost.py

```diff
@@ -91,7 +93,9 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        expand_missing_features=False,
```

Comment:
Yes, it seems that the only place where it makes sense to collapse hidden features is for text, which is a rare use-case for tree ensembles. So let's rename it to "collapse_missing_features" and disable by default?